src/cmd.js

```diff
@@ -145,6 +155,16 @@ function handleSequenceList(
 }
 
 export function executeCmd(cmd, dispatch, getState, loopConfig = {}) {
+  return executeCmdInternal(cmd, dispatch, dispatch, getState, loopConfig);
```

Comment:
Should we consider putting these four in an object at some point? It would be less error-prone than passing all four down.

Reply:
By a context object do you just mean one object with properties `cmd`, `dispatch`, `wrappedDisaptch`, etc... ?

What kinds of errors do you see happening with individual params? I didn't think this was a problematic pattern

Reply:
An error could be accidentally passing dispatch in place of wrappedDispatch. In my experience it's harder to mix up named attributes than it is to accidentally swap position arguments.

I think if there were 10 parameters here we would use an object. For now it's just a question of where you draw the line. Either way is fine.

Reply:
that's a good point. The parameter list could grow, and the 2 dispatches are definitely easy to mix up. i'll switch it  to use an object

Reply:
executeCmd (and   the internal helper) now take an object